obspy/signal/spectral_estimation.py

```diff
@@ -2184,23 +2207,27 @@ def __plot_coverage(self, ax):
         ax.autoscale_view()
 
 
-def get_nlnm():
+def get_nlnm(mod_fl):
     """
     Returns periods and psd values for the New Low Noise Model.
     For information on New High/Low Noise Model see [Peterson1993]_.
     """
-    data = np.load(NOISE_MODEL_FILE)
+    # The 'mod_fl' argument chooses between sesimic or infrasound noise models
+    # For information of the IDC infrasound noise models, see [Brown2012]
```

Comment:
Here, I think, the trailing underscore after citation is crucial for the Sphinx to properly interpret it as a link. Additionally, could you add a bib entry in the `obspy/misc/docs/source/bibliography` for that paper?

Reply:
Thanks - I added a smaller test file, added the underscores, and added the reference to the bibliography directory.